packages/helpers/product-helpers.ts

```diff
@@ -96,3 +96,17 @@ export function getProductIdFromGlobalId(globalProductId: string) {
 export function getProductVariantURI(variantId: string | number): string {
    return `gid://shopify/ProductVariant/${variantId}`;
 }
+
+export function getCategoriesFromTags(tags: string[]): string[] {
+   const tagsObj: Record<string, string> = {};
+   tags.forEach((tag) => {
+      const [name, val] = tag.split(/=|:/);
```

Comment:
I'm assuming that tags will never contain `:` or `=` except to separate the name and the value.

Reply:
@sterlinghirsh i don't know if you can say for certain, but yeah thats how i understood it.